docs/edgekv_cli.md

```diff
@@ -115,6 +115,7 @@ Commands:
 | download token `<tokenName> [options]` | Download an edgekv token. |
 | revoke token `<tokenName>` | Revoke an EdgKV access token. |
 | list auth-groups `[options]`| List the permission groups with EdgeKV Access. |
+| list groups `<environment> <nameSpace>` | List the groups for a given namespace in an Akamai environment. |
```

Comment:
Spoke to Sarah about this and it was agreed that the following would be better to avoid confusion with which groups are listed:
 | list groups `<environment> <nameSpace>` | List the data groups for a given namespace in an Akamai environment. |

Reply:
done